### ISSUE TITLE
Additional languages

### ISSUE THREAD

**Issue by GhisF** [mage-ai/mage-ai#1681]
Hi,

Do you think it will be possible in a near future to have have blocks using shell scripts (bash, zsh, fish...) or the Julia language ? 

Thank you

**Comment by wangxiaoyou1993** [comment]
Hi @GhisF , currently you can use bash script in scratchpad with `%%bash` header.
Do you want to run the pipeline with bash scripts? Could you elaborate your use cases here?

**Comment by GhisF** [comment]
Hi,
I am currently testing mage-ai as a tool to run pipeline, to process and analyse sequencing data for research.
It is basically a succession of steps using tools running in containers and custom scripts.
Shell scripts are used to manage files and job queues, Julia R and python scripts to extract useful information from the analysed data.
From %%bash it is possible to run all other script or container. But the possibility to use Julia code directly in a cell and render tables and plots like it is possible with Python, it could be a nice feature.


Mage-AI is a very nice tool, very easy to deploy and use.

**Comment by wangxiaoyou1993** [comment]
> Creating a Python scratchpad and using a %%bash is not very intuitive, And scratchpad are not executed during a pipeline run. Is it possible to add bash as an available language for all type of cells ?

Yes, definitely! Thanks for requesting the feature! We'll put this on our roadmap, and keep you updated on the progress of the feature.
